Jenkinsfile

```
@@ -1103,5 +1144,18 @@ pipeline {
                 }
             }
         }
+        stage("Performance Test - HIP") {
```

```suggestion
        stage("Performance Tests gfx90a") {
```
Beware that this stage (i.e. all the nested stages) will be run on MI200 nodes only, due to `when{}` at lines 1148-1150. This seems unusual. You may wish to move this check into the inner stage.

---

That was the plan to only run on 90a nodes initially.  @JehandadKhan should we update?

---

Yes, moving it to the inner block would still run it on gfx90a however, it would ensure that each stage specifies what arch it runs on.

---

Yes, for now lets stick with gfx90a and expand later on when we know how much time this takes and how much hardware resource we can spare.

---

[Q] Is this what we want? In other words, is this stage intended for performance testing on MI200 targets **_only_**?

---

I highly recommend staying consistent in the scripts (such as this one). This is important for maintainability.

Judge yourself. I consider this as resolved.